Production/test/unitTest.py

```diff
@@ -93,35 +93,10 @@ def defineTests(mytests, scenario, name, numevents, command, dataset, inputFiles
     mytests.append(Test(scenario,name,numevents,command,dataset,inputFilesConfig,nstart=0,nfiles=10,redir=""))
 
     # pre-defined tests
-    mytests.append(Test("Summer16v3","Summer16v3.GJets_HT-600ToInf_TuneCUETP8M1_13TeV-madgraphMLM-pythia8",numevents,command,inputFilesConfig="Summer16v3.GJets_HT-600ToInf_TuneCUETP8M1_13TeV-madgraphMLM-pythia8",nstart=0,nfiles=1))
```

Comment:
we have unit tests for GJets as well as ttbar in order to exercise the cleaned jet reclustering more thoroughly, so please add some for UL at some point (in addition to the signal and data ones, whenever those samples are available)